package.json

```diff
@@ -9,7 +9,10 @@
     "lint": "next lint"
   },
   "dependencies": {
+    "@google-cloud/firestore": "^7.1.0",
```

Comment:
why did you add these new packages by the way? i think it should work with just the firebase admin dependency 👀

Reply:
Can you try deleting the new dependencies in this and just keep the firebase-admin one. Run npm install to cleanup old deps and start the project again to see if it still works without all the extra deps.

Reply:
I was getting an error when I downloaded the changes sophie made to connect to firebase. It had said: "Cannot find module '@google-cloud/firestore/build/src/path" so I tried pulling from main again and running npm install and a couple other things but the only thing that worked was running npm install @google-cloud/firestore

Reply:
That's correct! I pulled your branch on my local and seems to work with just the `firebase-admin` dep. So please remove the duplicate ones in `package.json` and run `npm i` again. Check first if it works and proceed with the commit 🙇‍♀️

Reply:
Hey, can you try removing `node-modules` and re-running `npm i`. You can remove node_modules by running the following command
```
rm -rvf node_modules
```

Reply:
I did this ^^ and I'm still getting the same error

Reply:
we resolved this during huddle, thanks 🙇‍♀️